pkg/cmd/tracerunner.go

```diff
@@ -13,15 +13,51 @@ import (
 	"syscall"
 
 	"github.com/fntlnz/mountinfo"
+	"github.com/iovisor/kubectl-trace/pkg/tracejob"
 	"github.com/spf13/cobra"
 )
 
+const (
+	bpftrace = "bpftrace"
+	bcc      = "bcc"
+)
+
+var (
+	bpfTraceBinaryPath = "/bin/bpftrace"
+)
+
 type TraceRunnerOptions struct {
-	podUID             string
-	containerName      string
-	inPod              bool
-	programPath        string
-	bpftraceBinaryPath string
+	// The tracing system to use.
+	// tracer = bpftrace | bcc | perf
+	tracer string
+
+	// Selector (label query) that identifies entity to be traced.
+	// selector = label '=' value [',' labelN '=' valueN ...]
+	// Currently supported labels:
+	// - node
+	//   Select a node by name
+	// - pod
+	//   Select a pod by name
+	// - pod-uid
```

Comment:
we already handle namespaces elsewhere right?

Reply:
Yes, namespaces are handled globally by the `kubectl` command.